packages/colors/docs/sass.md

```diff
@@ -146,7 +145,7 @@ Define color variables
       90: #061f80,
       100: #051243,
     ),
-    'coolGray': (
```

Comment:
Were these gray re-namings intentional? E.g. `coolGray` to `cool-gray`?

Reply:
Yeah, we should do `param-case` in Sass, I think.

Reply:
Is that a breaking change?

Reply:
Yeah, I guess it would be. I'll add in support for both forms for now.